docs/sources/troubleshooting/troubleshoot-queries.md

```diff
@@ -22,3 +22,10 @@ like a bug or visualization issue in Grafana, it is almost always a problem with
 the data source response. Start by inspecting your panel query and response.
 
 For more information, refer to [Inspect a panel]({{< relref "../panels/inspect-panel.md" >}}).
+
+## My query is slow
+
+How many data points is your query returning? Queries returning lots of data points will be slow. Try this:
+- In Query options, limit the **Max data points** returned.
```

Comment:
Do we ever use e.g. italic to demarcate text from the UI, f.ex. _Query options_ (assuming that's text from the UI)? That would make it easier for me at least, to understand how to correlate the instructions here with the UI. It's the convention I follow myself when writing technical documents.

Reply:
Per the Documentation style guide and Microsoft style guide, UI text should be in bold. Thanks for pointing it out, I will fix this.

Reply:
@oddlittlebird OK that's good to know, main thing is there's a convention.